internal/pkg/flags/flags.go

```diff
@@ -19,6 +19,9 @@ const (
 	DefaultManifestRequeueErrInterval                                   = 2 * time.Second
 	DefaultManifestRequeueWarningInterval                               = 30 * time.Second
 	DefaultManifestRequeueBusyInterval                                  = 5 * time.Second
+	DefaultManifestRequeueJitterProbability                             = 0.10
```

Comment:
10% is too much. I think we agreed during the refinement of this issue, that the following params could be the default:
JitterProbablility: 2%,  JitterPercentage: 2%, always enabled.